autocomplete/definitions/global/tes3/rayTest.lua

```diff
@@ -1,54 +1,111 @@
 return {
 	type = "function",
-	description = [[Performs a ray test and returns various information related to the result(s). If `findAll` is set, the result will be a table of results, otherwise only the first result is returned.
+	description = [[Performs a ray test and returns various information related to the result(s). The ray test works by effectively shooting out a line, starting at `position` and pointing towards `direction`, and then checking to see which objects intersect that line.
+	
+Here is an overview of how some commonly used parameters will alter how `tes3.rayTest` checks for collisions:
+	1. `root`: Things that aren't a `child` of the specified `root` will be skipped. (If `root` is not provided, then nothing will be skipped by this process.)
 
-!!! tip Improving performance of rayTest
+	2. `ignore`: objects in this array will be skipped.
+
+	3. `maxDistance`: if specified, only objects within the specified distance will be checked.
 
-		1. Keep maximum size of objects reasonable, as well as triangle counts
+	4. `findAll`: If `true`, then all intersections will be returned. Otherwise, only the first intersection will be returned.
```

Comment:
Small nit, please make it consistent, so each sentence after the colon starts in either capital or lower case.